cupy/_core/raw.pyx

```diff
@@ -76,7 +76,7 @@ cdef class RawKernel:
         self.__name__ = name
 
     def __call__(self, grid, block, args, **kwargs):
-        """__call__(self, grid, block, args, *, shared_mem=0)
+        """__call__(self, grid, block, args, \*, shared_mem=0)
```

Comment:
To silence `/home/runner/work/cupy/cupy/cupy/_core/raw.pyx:79:46: W605 invalid escape sequence '\*'`

```suggestion
        """__call__(self, grid, block, args, \\*, shared_mem=0)
```